backend/build.gradle

```
@@ -32,37 +36,48 @@ dependencies {
     compileOnly 'org.projectlombok:lombok'
     annotationProcessor 'org.projectlombok:lombok'
 
+    // RESTDocs
```

```suggestion
    // Docs
```

라이브러리의 종류만 나눠놓으면 된다고 생각해, Docs 정도로도 의미 전달이 가능할 것 같습니다.